package-lock.json

```diff
@@ -0,0 +1,878 @@
+{
```

Comment:
Really appreciate setting up Standard for listing 🙇  We're trying to use yarn in Rails projects – as I understand it comes out of the box. There's a good reference for the [package.json in content-publisher](https://github.com/alphagov/content-publisher/blob/master/package.json).